doc/api/modules.md

```diff
@@ -171,7 +171,7 @@ The `.mjs` extension is reserved for [ECMAScript Modules][] which cannot be
 loaded via `require()`. See [Determining module system][] section for more info
 regarding which files are parsed as ECMAScript modules.
 
-## All together...
+## All together
```

Comment:
```suggestion
## All together&hellip;
```

Alternatively, and more correctly, but Microsoft Style would disagree?

Reply:
Yes, Microsoft would say to just avoid it altogether: "In general, don’t use an ellipsis (…) except in the situations described here or to indicate omitted code in technical content." https://docs.microsoft.com/en-us/style-guide/punctuation/ellipses